.gitignore

```diff
@@ -17,3 +17,5 @@ zwave-js-*.tgz
 *.map
 
 .tmp
+
+/**/package-lock.json
```

Comment:
That does not include the one in the root, correct?
Maybe being explicit would be better: `packages/*/package-lock.json`

Reply:
> That does not include the one in the root, correct?

Nope, going to edit it to: `packages/**/package-lock.json`

Reply:
one * is enough, don't need to target subdirectories of the packages